ProjectLighthouse/Administration/Maintenance/RepeatingTasks/DismissExpiredCasesTask.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LBPUnion.ProjectLighthouse.Database;
+using LBPUnion.ProjectLighthouse.Logging;
+using LBPUnion.ProjectLighthouse.Types.Entities.Moderation;
+using LBPUnion.ProjectLighthouse.Types.Logging;
+using LBPUnion.ProjectLighthouse.Types.Maintenance;
+using Microsoft.EntityFrameworkCore;
+
+namespace LBPUnion.ProjectLighthouse.Administration.Maintenance.RepeatingTasks;
+
+public class DismissExpiredCasesTask : IRepeatingTask
+{
+    public string Name => "Dismiss Expired Cases";
+    public TimeSpan RepeatInterval => TimeSpan.FromHours(4);
+    public DateTime LastRan { get; set; }
+
+    public async Task Run(DatabaseContext database)
+    {
+        List<ModerationCaseEntity> expiredCases =
```

Comment:
Does this account for timezones?

Reply:
it may not be an issue as long as the time in the case is set by the server, though DST may still be a problem

Reply:
Is it not stored in UTC?

Reply:
And even then it's not been an issue in any other aspect of the mod system such as case expiration so I would assume it'd be fine